src/main/java/appeng/integration/modules/inventorytweaks/InventoryTweaksModule.java

```diff
@@ -0,0 +1,41 @@
+
+package appeng.integration.modules.inventorytweaks;
+
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.Loader;
+
+import invtweaks.api.InvTweaksAPI;
+
+import appeng.integration.abstraction.IInvTweaks;
+
+
+public class InventoryTweaksModule implements IInvTweaks
+{
+	InvTweaksAPI api = null;
+
+	public InventoryTweaksModule()
+	{
+		try
+		{
+			api = (InvTweaksAPI) Class.forName( "invtweaks.forge.InvTweaksMod", true, Loader.instance().getModClassLoader() )
+					.getField( "instance" )
+					.get( null );
+		}
+		catch( Exception ex )
+		{
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00108)